.vaunt/config.yaml

```diff
@@ -0,0 +1,174 @@
+                        & labels in {'2024-tidb-docs-dash'}
+                        & emojis in ['THUMBS_UP']
+                        & reactors in ['rpaik', 'lilin90', 'qiancai', 'hfxsd', 'ran-huang', 'Oreoxmt','Yan-yyCL','dveeden']"
+                    value: 2
+    - action:
+          type: point
+          name: issue_contribution
+          triggers:
+              - trigger:
+                    actor: author
+                    action: issue_comment
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & emojis in ['THUMBS_UP']
+                        & reactors in ['rpaik', 'lilin90', 'qiancai', 'hfxsd', 'ran-huang', 'Oreoxmt','Yan-yyCL','dveeden']
+                        & labels in !['2024-tidb-docs-dash']"
+                    value: 1
+    - action:
+          type: point
+          name: pr_submitted
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & labels in !['2024-tidb-docs-dash']
+                        & labels in ['contribution']"
+                    value: 2
+    - action:
+          type: point
+          name: pr_merged
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-26 20:00:00
+                        & labels in !['2024-tidb-docs-dash']
+                        & labels in ['contribution']"
+                    value: 5
+    - action:
+          type: point
+          name: bonus_issue_comment
+          triggers:
+              - trigger:
+                    actor: author
+                    action: issue_comment
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & emojis in ['THUMBS_UP']
+                        & reactors in ['rpaik', 'lilin90', 'qiancai', 'hfxsd', 'ran-huang', 'Oreoxmt','Yan-yyCL','dveeden']
+                        & labels in ['tidb-docs-dash-bonus']"
+                    value: 5
+    - action:
+          type: point
+          name: bonus_pr
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition:
+                        "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-26 20:00:00
+                        & labels in ['contribution','tidb-docs-dash-bonus']"
+                    value: 20
+achievements:
```

Comment:
What happens if there are two contributors with the same score among the top five?

Reply:
Great question! This is an edge case that we can handle in a couple ways. Would you want both users to be awarded the achievement?

Reply:
I think in cases of a tie, we can award both users the achievement.

Reply:
Thanks - we will ensure this is how it is handled.